web-jehlomat/src/screens/SetNewPassword/Components/Buttons.tsx

```diff
@@ -0,0 +1,5 @@
+﻿import {styled} from "@mui/material";
+
+export const Buttons = styled('div')({
```

Comment:
At first glance, this component represents several components because of the naming. I would name it as ButtonsGroup

Reply:
Yupe, it suppose to be a presentational component. Will rename with more descriptive name.